Sources/Bow/Validation/Validation.swift

```diff
@@ -0,0 +1,475 @@
+import Foundation
+
+public class Validation<E> {
```

Comment:
The validation framework is a remnant from Funktionale that's in it's way out for not being lawful. I'd also not suggest not adding it.